examples/translator/proto/gen.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package proto
+
+//go:generate protoc -I=. --go_out=plugins=grpc,paths=source_relative:. translator.proto
```

Comment:
I'm confused, we need **both** the go_protobuf bazel rule **and** go-generate?

Reply:
As per the commit message, this allows the code to also be built directly with `go build ./translator.go`.  That was definitely helpful during the fighting-with-Bazel phase of development, but not strictly necessary now, so I'm on the fence as to whether to keep it.